### PR TITLE
Mariuszhussariush/ch7763/create embeddable additional cost calculator fixes

### PR DESCRIPTION
**Pull request by mariusz-hussariush**
styling and other final fixes

### CODE REVIEW

File: additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.css [hunk 1]
```diff
@@ -159,12 +199,48 @@ input[type='checkbox']:checked + label:after {
...
...CbHVyT3V0ZXIxIiBzdGREZXZpYXRpb249IjciLz48ZmVDb2xvck1hdHJpeCBpbj0ic2hhZG93Qmx1ck91dGVyMSIgcmVzdWx0PSJzaGFkb3dNYXRyaXhPdXRlcjEiIHZhbHVlcz0iMCAwIDAgMCAwIDAgMCAwIDAgMCAwIDAgMCAwIDAgMCAwIDAgMC4xIDAiLz48ZmVNZXJnZT48ZmVNZXJnZU5vZGUgaW49InNoYWRvd01hdHJpeE91dGVyMSIvPjxmZU1lcmdlTm9kZSBpbj0iU291cmNlR3JhcGhpYyIvPjwvZmVNZXJnZT48L2ZpbHRlcj48L2RlZnM+PGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj48cGF0aCBmaWxsPSIjRkVGRUZFIiBkPSJNLTMyMS0zMjJINTRWNzU0aC0zNzV6Ii8+PHJlY3Qgd2lkdGg9IjM1NCIgaGVpZ2h0PSI2MzgiIHg9Ii41IiB5PSIuNSIgZmlsbD0iI0ZGRiIgc3Ryb2tlPSIjMkU1QkZGIiBzdHJva2Utb3BhY2l0eT0iLjA4IiBmaWx0ZXI9InVybCgjYSkiIHJ4PSIxIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMzExIC02NikiLz48cGF0aCBmaWxsPSIjRkZGIiBzdHJva2U9IiMyRTM4NEQiIGQ9Ik0tMjk1LjUtMjAuNWgzMjR2NDloLTMyNHoiLz48dGV4dCBmaWxsPSIjMkUzODREIiBmb250LWZhbWlseT0iQXZlbmlyTmV4dC1SZWd1bGFyLCBBdmVuaXIgTmV4dCIgZm9udC1zaXplPSIxNiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTI5NiAtNDYpIj48dHNwYW4geD0iMTUiIHk9IjU1Ij5CdW5kZXNsYW5kIGF1c3fDpGhsZW48L3RzcGFuPjwvdGV4dD48ZyBzdHJva2U9IiMyRTM4NEQiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSIyIj48cGF0aCBkPSJNNi41IDYuMjEybDQuODMzLTQuNDY1TDYuNSA2LjIxMnptMCAwTDEuNjY3IDEuNzQ3IDYuNSA2LjIxMnoiLz48L2c+PC9nPjwvc3ZnPg==");
```
Comment (hariscustovic) [inline additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.css]: we could try doing this with cloudinary?

File: additionalcostcalculator/stencil.config.ts [hunk 2]
```diff
@@ -2,18 +2,24 @@ import { Config } from '@stencil/core';
...
-    {
-      type: 'www',
-      serviceWorker: null // disable service workers
+  },
+  {
+    type: 'docs-readme'
+  },
+  {
+    type: 'www',
+    serviceWorker: null, // disable service workers
```
Comment (hariscustovic) [inline additionalcostcalculator/stencil.config.ts]: comma can go

File: additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.tsx [hunk 3]
```diff
@@ -13,25 +13,53 @@ const InitialState = {
...
-  @State() purchasePrice: any = InitialState.purchasePrice
+  @State() purchasePrice: any;
   @State() withBrokerCommission: boolean = InitialState.withBrokerCommission
-  @State() brokerCommission: number = InitialState.brokerCommission
+  @State() brokerCommission: any = InitialState.brokerCommission
   @State() triggerAnimation: boolean = InitialState.triggerAnimation
-  @State() type: string = 'text';
+  @State() purchaseType: string = 'text';
+  @State() percentType: string = 'text';
+  @Element() el: HTMLElement;
```
Comment (hariscustovic) [inline additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.tsx]: Not that it matters much, but some lines end with ; and some don't. Would be cool if we could follow one style or the other.

File: additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.css (hunk 1 above)
Comment (mariusz-hussariush) [reply additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.css]: yup

File: additionalcostcalculator/stencil.config.ts (hunk 2 above)
Comment (mariusz-hussariush) [reply additionalcostcalculator/stencil.config.ts]: yup

File: additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.tsx (hunk 3 above)
Comment (mariusz-hussariush) [reply additionalcostcalculator/src/components/additional-cost-calculator/additional-cost-calculator.tsx]: done

**Pull request merged**